dev-ruby/vagrant_cloud/vagrant_cloud-2.0.1.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+USE_RUBY="ruby23 ruby24 ruby25"
+
+RUBY_FAKEGEM_EXTRADOC="CHANGELOG.md README.md"
+RUBY_FAKEGEM_GEMSPEC="${PN}.gemspec"
+RUBY_FAKEGEM_RECIPE_TEST="rspec3"
+RUBY_FAKEGEM_TASK_DOC=""
+
+inherit ruby-fakegem
+
+DESCRIPTION="Vagrant Cloud API Library"
+HOMEPAGE="https://github.com/hashicorp/vagrant_cloud"
+SRC_URI="https://github.com/hashicorp/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
```

Comment:
This too.

Reply:
I like to keep those, just preference, 15439 other ebuilds do the same.